src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/issuenav/DetailView.kt

```diff
@@ -0,0 +1,53 @@
+package com.atlassian.performance.tools.jiraactions.api.page.issuenav
+
+import com.atlassian.performance.seleniumjs.NativeExpectedCondition
+import com.atlassian.performance.seleniumjs.NativeExpectedConditions.Companion.and
+import com.atlassian.performance.seleniumjs.NativeExpectedConditions.Companion.or
+import com.atlassian.performance.seleniumjs.NativeExpectedConditions.Companion.presenceOfElementLocated
+import com.atlassian.performance.tools.jiraactions.api.page.isElementPresent
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import com.atlassian.performance.tools.jiraactions.api.webdriver.JavaScriptUtils
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions.elementToBeClickable
+import java.time.Duration
+
+class DetailView(
```

Comment:
Why do we want to expose those page objects in our API? Could we limit this lib to only provide actions?

Reply:
`internal` sounds good, we could expose them only when needed.
I tried it, but unfortunately, it has to be exposed for `IssueNavigatorPage` API:
![image](https://user-images.githubusercontent.com/1194704/227921393-08dee510-b15b-4504-9e82-d3c9ebb21d76.png)

If we hide/move `IssueNavigatorPage.waitForResults(results: IssueNavResultsView)`, then `IssueNavigatorPage` would lose its only non-deprecated API.

Reply:
Could we not deprecate the `waitForIssueNavigator()` method? It seems to be able to detect the view and wait for whatever is needed.

Reply:
The `UnknownView` impl irepresents a user, who has no list/detail view preferences and goes with whatever is currently selected. I thought it was a complication caused by old wrong assumptions and we don't need that anymore. Perhaps we still want to model such a user, but new API also allows that.